packages/@ourworldindata/grapher/src/core/Grapher.jsdom.test.ts

```diff
@@ -77,20 +77,31 @@ it("can get dimension slots", () => {
 it("an empty Grapher serializes to an object that includes only the schema", () => {
     expect(new Grapher().toObject()).toEqual({
         $schema: defaultGrapherConfig.$schema,
+
+        // ideally, selectedEntityNames is not serialised for an empty object
+        selectedEntityNames: [],
```

Comment:
This is unfortunate, but I'm not sure how to get rid of it. Although it's not nice, I don't think it has to be a blocker for this PR since it doesn't lead to buggy behaviour (afaik).